Elastic indices information (#480)
ElrondNetwork/docs.elrond.com: docs/sdk-and-tools/elastic-search.md

## Clone an Elasticsearch cluster

In order to have all the information about the Elrond chain in an Elasticsearch cluster (from genesis to current time) one has to copy all the data with a specific tool from an Elasticsearch cluster to another.
To get more information how to do this use the documentation from this [repository](https://github.com/ElrondNetwork/elrond-tools-go/tree/main/elasticreindexer).

## Elasticsearch indices 

> An observing-squad with the elastic indexer enabled will save data in different indices. This data is used to solve a lot of use cases. An example is to fetch all the 

Reviewer: ```suggestion
An observing-squad with the elastic indexer enabled will save data in different indices. This data is used for multiple use cases. An example is to fetch all the 
```

transactions that belong to an address or to display all the address sorted based on the EGLD balances.

> Saved data is grouped into indices.

Reviewer: You can delete it, since you've already mentioned it before : 
>An observing-squad with the elastic indexer enabled will save data in different indices

```suggestion
```

Author: done

In order to have all the information about the Elrond chain in an Elasticsearch cluster (from genessis to current time) one has to copy all the data with a specific tool from an Elasticsearch cluster to another.
To get more information how to do this use the documentation from this [repository](https://github.com/ElrondNetwork/elrond-tools-go/tree/main/elasticreindexer).

## Elasticsearch indices 

> An observing-squad with the elastic indexer enabled will save data in different indices. This data is used to solve a lot of use-cases. An example is to fetch all the 

Author: done

> transactions that belongs to an address or to display all the address sorted based on the EGLD balances.

Author: done

> All the information that is saved in grouped in indices.

Author: done

Each entry in an Elasticsearch index will have a format similar to this:

```
{
    "_id": "..."
    "_source": {
      ...
    }
}
```

| Name                                                | Description                                                                 |
|-----------------------------------------------------|-----------------------------------------------------------------------------|

> | [rating](/sdk-and-tools/indices/rating)             | Contains information about the validators rating in every epoch.            |

Author: done

| [accountshistory](/)                                         | Contains historical information about the address balances.                 |
| [receipts](/)                                                | Contains all the generated receipts.                                        |
| [scresults](/)                                               | Contains all the generated smart contract results.                          |
| [accountsesdt](/)                                            | Contains the addresses' ESDT balances.                                      |
| [accountsesdthistory](/)                                     | Contains historical information about the address ESDT balances.            |
| [epochinfo](/)                                               | Contains the accumulated fees and the developer fees grouped by epochs.     |
| [scdeploys](/)                                               | Contains details about all the deployed smart contracts.                    |
| [tokens](/)                                                  | Contains all the created ESDT tokens.                                       |

> | [tags](/)                                                    | Contains the NFT' tags.                                                     |

Reviewer 2: NFTs' tags*

Author: done
